hvantk/utils/gene_sets.py

```
@@ -1,18 +1,585 @@
+    --------
+    >>> collection = load_marker_genes("markers.tsv", cluster_column="cell_type")
+    """
+    import pandas as pd
+
+    marker_file = Path(marker_file)
+    logger.info(f"Loading marker genes from {marker_file}")
+
+    df = pd.read_csv(marker_file, sep="\t")
+
+    if cluster_column not in df.columns:
+        raise ValueError(f"Column '{cluster_column}' not found in {marker_file}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)